tools/docker/builder/Dockerfile

```diff
@@ -21,6 +21,12 @@ RUN apt-get update && apt-get install -y \
 	libzmq3-dev \
 	python \
 	python-yaml \
+	net-tools \
```

Comment:
This is a builder dockerfile - please don't change it, you should use the runner dockerfile for testing